src/stories/Button.stories.mdx

```diff
@@ -82,6 +82,78 @@ export const Template = args => <Button {...args} />;
 
 <br />
 
+### Rounded
```

Comment:
We don't need a rounded button. The consumer can just override the button styles as specified in the docs. We don't want this thing to cater for all possible scenarios. We want to keep it lean and as unopinionated as possible.